patches/installer/0007-Add-tools-folder-to-crossgen-exclusions-for-MS.NET.B.patch

```diff
@@ -0,0 +1,52 @@
+From 2cdd560b4029623d27a7133b2651a76c2b209794 Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Thu, 8 Oct 2020 22:19:34 +0000
+Subject: [PATCH] Add tools folder to crossgen exclusions for
+ MS.NET.Build.Extensions
+
+---
```

Comment:
Why? Should the upstream repo be changed to not produce a `tools` folder instead, is this a diff vs. the Microsoft-built package? (File issue for followup with more info if it exists?)

Reply:
Removed the patch.  There was an issue in how sdk builds MS.NET.Build.Extensions.  I made a change there.